index.html

```diff
@@ -0,0 +1,74 @@
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title> CLARENCE PORTFOLIO</title>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300&display=swap" rel="stylesheet">
+</head>
+
+
+<body>
+  
+  <header class="header" id="header">
+    <a href="#">CLARENCE</a>
+    <nav>
+      <span class="open" id="menu"></span>
+       <img src="images/Icon - Menu.png" alt="menu bar" class="icon-container">
+      
+    </nav>
```

Comment:
Your toolbar section looks great. Kindly ensure that it's as close to the design as possible. It's height on your design seems larger than that of the design.
Also, endeavor to remove the little margins around it.

Your design:
![image](https://user-images.githubusercontent.com/20554583/235294770-1aaab98b-4a69-4c2c-94cf-1aca8c19bdc2.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/235294687-7bf3174b-cd98-4291-be19-8fa8b5f40526.png)